### PR TITLE
Updates the barsigns!

### PR DESCRIPTION
**Pull request by Supermichael777**
Bar-sign memes were dated/lame. This "fixes" that.
the Emergency rum party had a joke about Scardy's autism rules that is no longer true, and the Net had so much potential for good puns yet had a lame one anyway.

### CODE REVIEW

File: code/game/objects/structures/barsigns.dm [hunk 1]
```diff
@@ -246,7 +246,7 @@
 /datum/barsign/emergencyrumparty
 	name = "The Emergency Rum Party"
 	icon = "emergencyrumparty"
-	desc = "Still serving drinks that were banned years ago."
+	desc = "Recently relicenced after a long closure."
```
Comment (xxalpha) [inline code/game/objects/structures/barsigns.dm]: relicensed

Comment (bgobandit) [inline code/game/objects/structures/barsigns.dm]: british spelling pls

File: code/game/objects/structures/barsigns.dm [hunk 2]
```diff
@@ -276,7 +276,7 @@
 /datum/barsign/thenet
 	name = "The Net"
 	icon = "thenet"
-	desc = "The sea of drinkformation." //you couldn't come up with something better?
+	desc = "You just seem to get caught up in it for hours." //you couldn't come up with something better?
```
Comment (Anonus) [inline code/game/objects/structures/barsigns.dm]: This IS better, right?

File: code/game/objects/structures/barsigns.dm (hunk 1 above)
Comment (duncathan) [inline code/game/objects/structures/barsigns.dm]: American spelling is standard throughout the codebase and game. Make it "relicensed."

**Comment by Niknakflak** [comment]
This effects non of the descriptions I made for the original signs so I don't care :+1:

File: code/game/objects/structures/barsigns.dm (hunk 1 above)
Comment (RemieRichards) [inline code/game/objects/structures/barsigns.dm]: @duncathan that's not strictly true.
There's no rule on whether you use the wrong one (american) or the correct one (british)

**Comment by Supermichael777** [comment]
changed because honestly fuck why not.

**Pull request merged**